airflow/contrib/executors/mesos_executor.py

```diff
@@ -42,6 +44,10 @@ def __init__(self,
     def registered(self, driver, frameworkId, masterInfo):
         logging.info("AirflowScheduler registered to mesos with framework ID %s", frameworkId.value)
 
+        if conf.getboolean('mesos', 'CHECKPOINT') and conf.get('mesos', 'FAILOVER_TIMEOUT'):
+            with open(FRAMEWORK_ID_PATH, mode="w+") as f:
+                f.write(frameworkId.value)
```

Comment:
The ID of the framework is stored on the local disk. This has the drawback that re-registering only works while the Scheduler process is always started on the same node.

The ID could also be stored in the database. From a quick look at the schema, there is no table for storing arbitrary information. Should I add a new one for this purpose?